### PR TITLE
added example of error response for numbers

### PR DESCRIPTION
**Pull request by jmulford-bw**
## For the Committer

⚠️ Ensure that for this repo (**bandwidth.github.io**) that the pull request change is opened against the branch `stop-gap-v2`

### CODE REVIEW

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-DX-1403.s3-website-us-east-1.amazonaws.com/

**Comment by DX-Bandwidth** [comment]
Preview Changes at:
        http://bw-dev-DX-1403.s3-website-us-east-1.amazonaws.com/

File: numbers/error.md [hunk 1]
```diff
@@ -1,3 +1,33 @@
+Errors that may arise when performing tasks like searching for phone numbers or ordering phone numbers have specific error codes associated with them. These error codes and a human readable description are listed on this page.
+
+XML tags `<Code>` and `<Description>` will be included in the response body. Refer to the API endpoints for specific formats. Below is an example for a response when trying to order a malformed phone number.
```
Comment (BryanZWu) [inline numbers/error.md]: Maybe a link here to the API? I know that you can't link to all of the endpoints but maybe to the devdocs or something. Just a suggestion

**Review by BryanZWu — APPROVED** [review]

**Review by ajrice6713 — APPROVED** [review]

File: numbers/error.md (hunk 1 above)
Comment (jmulford-bw) [reply numbers/error.md]: Ez fix, works for me. Done

**Pull request merged**